### PR TITLE
feat: adds sandbox env config

### PR DESCRIPTION
**Pull request by cesarenaldi**

### CODE REVIEW

File: packages/react/src/environments.ts [hunk 1]
```diff
@@ -75,6 +75,32 @@ export const development: EnvironmentConfig = {
   handleResolver: (handle) => `${handle}.test`,
 };
 
+/**
+ * The sandbox environment configuration
+ *
+ * This is the environment to be used when you develop and you need to experiment with custom collect/follow modules.
```
Comment (desfero) [inline packages/react/src/environments.ts:81]: Do the lens-sdk support custom collect/follow modules as i quickly found inside `createCollectRequest` the invariant that's throw for unknown collect modules

**Review by desfero — APPROVED** [review]

**Comment by desfero** [comment]
And seems changeset is missing

File: packages/react/src/environments.ts (hunk 1 above)
Comment (cesarenaldi) [reply packages/react/src/environments.ts:81]: It does support returning unknown collect and follow settings data on publications/profiles.... but not configuration of them.

Comment (desfero) [reply packages/react/src/environments.ts:81]: got it, so there is no way latter to collect that publication through lens-sdk? Should go though some custom logic, right?

Comment (desfero) [reply packages/react/src/environments.ts:81]: Maybe we could think for lens-sdk v2 about some way of collecting these sort of posts as could be a blocker for someone to use lens-sdk?

Comment (cesarenaldi) [reply packages/react/src/environments.ts:81]: We could definitely enable more of these. Right now creating custom follow/collect module is not a very common occurrence and requires some coordination in terms of review, audit, whitelisting them. So it's not something happening very often.

**Pull request merged**